Dockerfile

```diff
@@ -1,20 +1,10 @@
-FROM python:3
+FROM python:3-alpine
 
-ENV TERRAFORM_VERSION=0.9.2
+ENV TERRAFORM_VERSION=0.9.4
```

Comment:
Hmmm what's that???  A terraform version upgrade???  Not mentioned in a commit??? 🤓